manifests/dispatch_router.pp

```diff
@@ -34,19 +34,12 @@
 
   contain qpid::router
 
-  # SSL Certificate Configuration
-  class { 'certs::qpid_router':
-    require => Class['qpid::router::install'],
-  }
-  ~> qpid::router::ssl_profile { 'client':
-    ca   => $certs::ca_cert,
-    cert => $certs::qpid_router::client_cert,
-    key  => $certs::qpid_router::client_key,
-  }
-  ~> qpid::router::ssl_profile { 'server':
+  include certs::qpid_router
```

Comment:
Previously there was a notify that if a certificate changed, qpid would be restarted. Not 100% sure it worked, but after this change it certainly is no longer there. While I prefer the `include` over `class { '...': }`, I'd ask you to add `subscribe => Class['certs::qpid_router']` to the `ssl_profile`. That way you also guarantee that the certs that the profile uses are present prior to starting qpid router.